src/main/java/racingCar/domain/Position.java

```diff
@@ -0,0 +1,21 @@
+package racingCar.domain;
+
+public class Position {
+    private static final int INITIAL_POSITION = 0;
+
+    private int cur;
+
+    public Position() {
+        cur = INITIAL_POSITION;
+    }
```

Comment:
```suggestion
    private int cur = 0;
```

외부에서 INITIAL_POSITION을 사용하지 않고 있으니, 위와 같이 초기화 하는건 어떨까요?